opensearch_py_ml/ml_commons/rag_pipeline/rag/opensearch_class.py

```diff
@@ -0,0 +1,127 @@
+# opensearch_class.py
```

Comment:
we need to add apache 2.0 license header to every files.

Reply:
got you!